core/validation/specific_validation/towns_fund_round_four/validate.py

```diff
@@ -354,7 +353,7 @@ def validate_psi_funding_gap(workbook: dict[str, pd.DataFrame]) -> list["Generic
 
 def validate_funding_spent(workbook: dict[str, pd.DataFrame]) -> list["GenericFailure"] | None:
     """Validates that total reported funding spent on the sheet is less than the amount specified as allocated in a
-    separate sheet stored on S3.
+    separate sheet stored in the repo.
```

Comment:
what is "the repo" in this instance? Is this a bit vague?

Reply:
Good point, added a bit more clarity here